tests/unittests/tests-uri_parser/tests-uri_parser.c

```diff
@@ -67,17 +70,21 @@ typedef struct {
       scheme, userinfo, host, port,
       path, query, expected return value)
 */
-static const validate_t validate_uris[27] = {
+static const validate_t validate_uris[28] = {
```

Comment:
Just remove the static size, so this needs not be adapted everytime there is a new test case?
```suggestion
static const validate_t validate_uris[] = {
```